src/components/radio/Radio.tsx

```diff
@@ -0,0 +1,46 @@
+import { Component, createSignal, Show } from 'solid-js'
+
+import StyledIcon from './StyledIcon'
+import StyledInput from './StyledInput'
+import StyledLabel from './StyledRadio'
+
+export type IRadioProps = {
```

Comment:
Can we name this just `RadioProps` to match the other component prop names?

Reply:
Yeah, we can name it `RadioProps`, we were using the `I` convention in our recent projects so I added the `I`. I'll remove it if we don't use that convention.